mongo/datadog_checks/mongo/mongo.py

```diff
@@ -175,9 +175,18 @@ def _build_metric_list_to_collect(self):
 
         return metrics_to_collect
 
+    def _refresh_replica_role(self):
+        if (
```

Comment:
I think this needs some logging when refresh_role is true but the other conditions are not met

Reply:
Updated! The api client should never be null now so I didn't log when it is

Reply:
Ok, so I ended up removing the option as mentioned here: https://github.com/DataDog/integrations-core/pull/10843#issuecomment-995928510, but I added a log when the conditions are met.